packages/focus-lock/tests/focus-lock.test.tsx

```diff
@@ -12,11 +12,32 @@ test("focuses an element on render", () => {
       </FocusLock>
     )
   }
-  const utils = render(<Component />)
-  const input = utils.getByTestId("input")
+  render(<Component />)
+  const input = screen.getByTestId("input")
   expect(input).toHaveFocus()
 })
 
+// placeholder test until we figure out how to deal with react-focus-lock
+// being
```

Comment:
Being what?! The suspense is killing me! 😁

Reply:
uh, ehrm, good question, next question 👀 I'll remove it